include/net/nrf_cloud.h

```diff
@@ -260,7 +260,11 @@ enum nrf_cloud_topic_type {
 	 *  messages into a single message that will be unwrapped and re-published to the
 	 *  message topic in the nRF Cloud backend.
 	 */
-	NRF_CLOUD_TOPIC_BULK
+	NRF_CLOUD_TOPIC_BULK,
+	/** Endpoint used to publish binary data to nRF Cloud for certain services.
+	 *  One example is LOG_OUTPUT_DICT formatted logs.
```

Comment:
In `zephyr/include/zephyr/logging/log_output.h`.  But since that's obscure, I should make the same point using the public Kconfig variable that enables dictionary logging instead.